src/EditorFeatures/Core.Wpf/InlineRename/UI/Adornment/RenameFlyout.xaml

```diff
@@ -44,7 +44,7 @@
                     Text="{Binding IdentifierText, UpdateSourceTrigger=PropertyChanged}" 
                     GotFocus="IdentifierTextBox_GotFocus"
                     HorizontalAlignment="Stretch" 
-                    PreviewKeyDown="IdentifierTextBox_KeyDown"/>
+                    PreviewKeyDown="IdentifierTextBox_PreviewKeyDown"/>
```

Comment:
why this cahnge?

Reply:
Just rename to align with wpf event name

Reply:
oh.  d'oh.  i thought we were hooking up to a different event.  thanks :)